app/models/nic/bmc.rb

```diff
@@ -12,15 +12,15 @@ def virtual
     end
     alias_method :virtual?, :virtual
 
-    attr_exportable :provider, :username, :password => ->(nic) { nic.decrypt_field(nic.password) }
+    attr_exportable :provider, :username, :password
```

Comment:
The Encryptable module automatically defines the `password` getter with a decryption routine, so this was redundant (https://github.com/theforeman/foreman/blob/develop/app/models/concerns/encryptable.rb#L32), overwriting the default Rails getter which would return the DB contents. The alias_method I added then renames the `password` getter to `password_unredacted`.

There was test coverage already at https://github.com/theforeman/foreman/blob/develop/test/unit/nics/bmc_test.rb#L29